wp-content/themes/core/twig-components/blocks/wysiwyg/wysiwyg.twig

```diff
@@ -1,13 +1,13 @@
 {# Block: WYSIWYG #}
 
 <div class="s-content">
-	<div class="g-row g-row--center g-row--col-3--min-full"
-			 data-depth="0"
-			 data-name="columns"
-			 data-livetext>
+	<div data-depth="0"
```

Comment:
we can drop these data attributes, yes?

Reply:
Sure can and they will be as they are translated over from this now legacy directory back to components in removing twig, just hadn't gotten to it yet ; )